### ISSUE TITLE
Documentation does not describe PageLayout or PageMode

### ISSUE THREAD

**Issue by hackerb9** [pikepdf/pikepdf#327]
I had thought perhaps pikepdf could not change the initial PageLayout or PageMode since it does not appear to be documented anywhere. Other PDF libraries have special functions just for changing these important settings, but it turns out it is possible to do in pikepdf.

## Example

```python
from pikepdf import Pdf, Dictionary, Name
pdf=Pdf.open('input.pdf')
pdf.root.PageLayout=Name('/SinglePage')
pdf.root.PageMode=Name('/FullScreen')
pdf.save('output.pdf')
```

## PDF Reference 1.7

According to PDF Reference 1.7, `PageLayout` and `PageMode` are optional entries in the catalog dictionary. They are name objects which can take a small set of string values. Support can vary depending upon which PDF viewer is used.

## PageLayout

PageLayout specifies the page layout to be used when the document is opened. 

|Value|Meaning|
|---|---|
|/SinglePage|Display one page at a time (default)|
|/OneColumn|Display the pages in one column|
|/TwoColumnLeft|Display the pages in two columns, with odd-numbered pages on the left|
|/TwoColumnRight|Display the pages in two columns, with odd-numbered pages on the right|
|/TwoPageLeft|Display the pages two at a time, with odd-numbered pages on the left|
|/TwoPageRight|Display the pages two at a time, with odd-numbered pages on the right|

## PageMode

PageMode specifies how the document should be displayed when opened.

|Value|Meaning|
|---|---|
|/UseNone|Neither document outline nor thumbnail images visible (default)|
|/UseOutlines|Document outline visible|
|/UseThumbs|Thumbnail images visible|
|/FullScreen|Full-screen mode, with no menu bar, window controls, or any other window visible|
|/UseOC|Optional content group panel visible|
|/UseAttachments|Attachments panel visible|

**Comment by jbarlow83** [comment]
Would you mind submitting a PR to update the documentation where you think this should go?

I'm open to other viewpoints but I don't see a strong case for creating a special API for this. The PDF spec is really simple in this area. I try to do that when getting the PDF spec right is a significant undertaking, e.g. image handling.

**Comment by hackerb9** [comment]
I agree that a special API is unnecessary, as long as it is well documented so people don't mistakenly think PikePDF is deficient.

I was hoping you'd be able to paste my suggestion nearly verbatim into the docs. I have not had a chance to learn how PikePDF's documentation system works. If you do not have the spare cycles to do it, I understand and will try to do it when next I get some free time.

As for where it makes the most sense to insert it. It seems like it should be a "topic", on the same level as Pages. However, I admit, I had looked for it first in the API section, since that's where PageLayout and PageMode were for other libraries. I also checked the Tutorial and examples, figuring that the initial view is very basic functionality.

Are there many other magic variables like this which users will want to know about but are not handled in the API? Perhaps there needs to be a whole chapter dedicated to them?
